packages/web-react/src/components/Modal/ModalDialog.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ModalDialogProps } from '../../types';
+import { Button } from '../Button';
+import { Modal, ModalHeader, ModalDialog, ModalBody, ModalFooter } from '.';
+
+const meta: Meta<typeof ModalDialog> = {
+  title: 'Components/Modal',
+  component: ModalDialog,
+  argTypes: {
+    maxHeightFromTabletUp: {
+      control: 'text',
+    },
+    preferredHeightOnMobile: {
+      control: 'text',
+    },
+    preferredHeightFromTabletUp: {
+      control: 'text',
+    },
+  },
+  args: {
+    maxHeightFromTabletUp: '',
+    preferredHeightOnMobile: '',
+    preferredHeightFromTabletUp: '',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof ModalDialog>;
+
+const ModalWithHooks = (
+  args: React.JSX.IntrinsicAttributes &
```

Comment:
@literat Is this correct? I could only use ModalFooterProps in other components, but here it needs more.

Reply:
It seems to be correct.